index.html

```diff
@@ -6,6 +6,6 @@
     <title>Potionfy</title>
 </head>
 <body>
-    
+    <h1>Welcome to Potionfy. Enjoy your stay.</h1>
```

Comment:
could u remove enjoy ur stay and make the h1 red

Reply:
got it